client/src/main/scala/org/apache/celeborn/client/commit/MapPartitionCommitHandler.scala

```diff
@@ -107,6 +109,40 @@ class MapPartitionCommitHandler(
     dataCommitSuccess
   }
 
+  override def getUnCommitPartitionRequests(
+      shuffleId: Int,
+      shuffleCommittedInfo: ShuffleCommittedInfo): mutable.Set[PartitionLocation] = {
+    shuffleCommittedInfo.unCommitPartitionLocations.asScala.filterNot { partitionLocation =>
+      shuffleCommittedInfo.handledCommitPartitionLocations.contains(partitionLocation) &&
+      this.isPartitionInProcess(shuffleId, partitionLocation.getId)
+    }
+  }
+
+  override def incrementInFlightNum(
+      shuffleCommittedInfo: ShuffleCommittedInfo,
+      workerToRequests: Map[WorkerInfo, collection.Set[PartitionLocation]]): Unit = {
+    workerToRequests.foreach {
+      case (_, partitions) =>
+        partitions.groupBy(_.getId).foreach { case (id, _) =>
+          val atomicInteger = shuffleCommittedInfo
+            .partitionInFlightCommitRequestNum
+            .computeIfAbsent(id, (k: Int) => new AtomicInteger(0))
+          atomicInteger.incrementAndGet()
+        }
+    }
```

Comment:
Miss `super.incrementInFlightNum` in the end of this method?

Reply:
as map partition only need wait the single partition finished, so I remove it this time.